contracts/tokenomics/maker/tests/integration.rs

```diff
@@ -118,6 +117,33 @@ fn instantiate_contracts(
         )
         .unwrap();
 
+    let escrow_fee_distributor_contract = Box::new(ContractWrapper::new_with_empty(
+        astroport_escrow_fee_distributor::contract::execute,
+        astroport_escrow_fee_distributor::contract::instantiate,
+        astroport_escrow_fee_distributor::contract::query,
+    ));
+
+    let escrow_fee_distributor_code_id = router.store_code(escrow_fee_distributor_contract);
+
+    let init_msg = astroport_governance::escrow_fee_distributor::InstantiateMsg {
+        owner: owner.to_string(),
+        astro_token: astro_token_instance.to_string(),
+        voting_escrow_addr: "voting".to_string(),
+        emergency_return_addr: owner.to_string(),
+        start_time: 0,
+    };
+
+    let governance_instance = router
+        .instantiate_contract(
+            escrow_fee_distributor_code_id,
+            owner.clone(),
+            &init_msg,
+            &[],
+            "Astroport escrow fee distributor",
+            None,
+        )
+        .unwrap();
+
```

Comment:
Maybe we should add a test that will display the sending of tokens, to make sure that the commission from transactions is actually credited from Maker to Governance_Distributor_Fee.

Reply:
We already have it in lines 612-617